lib/utils/note-utils.js

```diff
@@ -9,29 +11,17 @@
  */
 const noteTitleAndPreviewRegExp = /\s*(\S[^\n]*)\s*(\S[^\n]*)?/;
 
-/**
- * Matches the title and excerpt in a note's content skipping opening # from title
- *
- * Both the title and the excerpt are determined as
- * content starting with something that isn't
- * whitespace and leads up to a newline or line end
- *
- * @type {RegExp} matches a title and excerpt in note content skipping opening #'s from title and preview
- */
-const noteTitleAndPreviewMdRegExp = /(?:#{0,6}\s+)?(\S[^\n]*)\s*(?:#{0,6}\s+)?(\S[^\n]*)?/;
-
 /**
  * Returns the title and excerpt for a given note
  *
  * @param {Object} note a note object
  * @returns {Object} title and excerpt (if available)
  */
 export const noteTitleAndPreview = note => {
-  const content = note && note.data && note.data.content;
+  let content = (note && note.data && note.data.content) || '';
+  content = isMarkdown(note) ? removeMarkdown(content) : content;
```

Comment:
What do you think about setting the [`stripListLeaders`](https://www.npmjs.com/package/remove-markdown#usage) option to false?

<img width="318" alt="screen shot 2018-11-09 at 19 01 26" src="https://user-images.githubusercontent.com/555336/48256238-56d43880-e452-11e8-99c8-e425228a9ac8.png">

It might actually be confusing if we strip them out.

Reply:
Yeah, I think that is a good idea.